packages/lib/modules/tokens/TokenRow/TokenRow.tsx

```diff
@@ -188,7 +188,7 @@ export default function TokenRow({
       if (customUsdPrice) {
         setUsdValue(bn(customUsdPrice).times(value).toString())
       } else if ((isBpt || isNestedBpt) && pool) {
-        setUsdValue(usdValueForBpt(address, chain, value))
+        setUsdValue(usdValueForAddress(address, chain, value))
```

Comment:
Is this only for BPT's or will it work for any address? what does the address refer to here?

Reply:
it will work for any token address, i will rename